### PR TITLE
Tweak: Use template literals in Elementor filter names

### PR DESCRIPTION
**Pull request by ramiy**
## PR Checklist
- [x] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Use curly braces around the filter name while declaring variables as part of the hook name.

### CODE REVIEW

File: assets/dev/js/editor/controls/wp_widget.js [hunk 1]
```diff
@@ -50,7 +50,9 @@ ControlWPWidgetItemView = ControlBaseDataView.extend( {
 					}
 				}
 
-				elementor.hooks.doAction( 'panel/widgets/' + self.model.get( 'widget' ) + '/controls/wp_widget/loaded', self );
+				var widgetType = self.model.get( 'widget' );
```
Comment (matipojo) [inline assets/dev/js/editor/controls/wp_widget.js:53]:
```suggestion
				const widgetType = self.model.get( 'widget' );
```

File: assets/dev/js/editor/regions/panel/pages/editor/commands/open.js [hunk 2]
```diff
@@ -10,14 +10,16 @@ export class Open extends CommandBase {
 			$e.route( this.component.getDefaultRoute(), args );
 		}
 
-		// BC: Run hooks after the route render's the view.
-		const action = 'panel/open_editor/' + args.model.get( 'elType' );
+		// BC: Run hooks after the route render's the view
+
+		const elementType = args.model.get( 'elType' );
+		const widgetType = args.model.get( 'widgetType' );
```
Comment (matipojo) [inline assets/dev/js/editor/regions/panel/pages/editor/commands/open.js:16]:
```suggestion
		const elementType = args.model.get( 'elType' ),
			widgetType = args.model.get( 'widgetType' );
```

**Review by matipojo — CHANGES_REQUESTED** [review]

File: assets/dev/js/editor/controls/wp_widget.js (hunk 1 above)
Comment (ramiy) [reply assets/dev/js/editor/controls/wp_widget.js:53]: @matipojo 
This was my initial thought, but the entire file uses `var` instead of `consts`.

**Review by matipojo — APPROVED** [review]

**Review by matipojo — CHANGES_REQUESTED** [review]
/home/runner/work/elementor/elementor/assets/dev/js/editor/controls/wp_widget.js
  55:46  error  Expected space(s) after '${'  template-curly-spacing
  55:58  error  Expected space(s) before '}'  template-curly-spacing

/home/runner/work/elementor/elementor/assets/dev/js/editor/elements/views/base.js
  50:58  error  Expected space(s) after '${'  template-curly-spacing
  50:71  error  Expected space(s) before '}'  template-curly-spacing

/home/runner/work/elementor/elementor/assets/dev/js/editor/regions/panel/pages/editor/commands/open.js
  19:48  error  Expected space(s) after '${'  template-curly-spacing
  19:61  error  Expected space(s) before '}'  template-curly-spacing
  22:48  error  Expected space(s) after '${'  template-curly-spacing
  22:61  error  Expected space(s) before '}'  template-curly-spacing
  22:63  error  Expected space(s) after '${'  template-curly-spacing
  22:75  error  Expected space(s) before '}'  template-curly-spacing

/home/runner/work/elementor/elementor/assets/dev/js/frontend/elements-handlers-manager.js
  137:61  error  Expected space(s) after '${'  template-curly-spacing
  137:74  error  Expected space(s) before '}'  template-curly-spacing
  141:62  error  Expected space(s) after '${'  template-curly-spacing
  141:74  error  Expected space(s) before '}'  template-curly-spacing

**Pull request closed without merging**